.github/pull_request_template.md

```diff
@@ -0,0 +1,14 @@
+## Summary
+Fixes #{ISSUE}
+
+### Time to review: __x mins__
+
+## Changes proposed
+> What was added, updated, or removed in this PR.
+
+## Context for reviewers
+> Testing instructions, background context, more in-depth details of the implementation, and anything else you'd like to call out or ask reviewers. Explain how the changes were verified.
```

Comment:
Cool this feels much more flexible than "Instructions to review", great addition